docs/themes/README.md

```diff
@@ -52,6 +52,16 @@ $interactive-01;
 $interactive-02;
 ```
 
+#### Feature flags
+
+You can disable the CSS custom property and theme output using a feature flag by writing the following Sass code before importing:
+
+```scss
+$security--feature-flags: (
+  themes: false,
+);
+```
```

Comment:
I'm torn on this - Consumers still have the ability to output CSS custom properties by enabling Carbon's experimental flag, but this refers directly to the transformations that we currently apply to support CSS theming. Perhaps it needs to be something like `css-themes`

Reply:
> this refers directly to the transformations that we currently apply to support CSS theming.

For sure! But I think this is confusing as-is.

Definitely think it should be renamed, but I wonder if `css-themes` is enough. A lot of times folks are coming to this library having already looked at Carbon's documentation for one reason or another, OR they use this library and *we* refer them back to Carbon's docs in various places, from our own docs. So calling it `themes` or `css-themes` with it set to `false` + the ability to toggle on/off is not clearly separating it from that existing Carbon functionality.

I would name it something like `security-custom-theme-properties` or something similar that specifically references "security" (delineate us from Carbon), "custom properties" (scope), and "theme" (scope). However you want to arrange that, I'm open to... but it really needs to completely describe the scope.

And then in addition to the name, I would update this documentation to specifically call out the different between toggling these custom properties versus Carbon's existing functionality. I would describe how this feature flag does/does not interact with Carbon's feature flags.

Reply:
I do like the idea of namespacing our feature flags to differentiate from Carbon - Currently, this is the naming for our feature flags:

```scss
(css-gridish: true, platform: true, ibm-type: true)
```

And some of Carbon's:

```scss
(ui-shell: true, grid: true, grid-columns-16: false, grid--fallback: false, enable-css-custom-properties: false)
```

I can apply some namespacing to this feature flag for now, but we'll need to create a follow-up issue to accommodate that naming for consistency amongst the others if this is the route we choose to go down, as this would constitute a breaking change.